src/components/TestButtons/index.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+
+import { Button } from 'antd';
+
+import Grid from 'antd/lib/card/Grid';
+import './styles.scss';
+import SnippetAccessModel from '../../model/SnippetAccessModel';
+import SnippetModel from '../../model/SnippetModel';
+import UserInfoModel from '../../model/UserInfoModel';
+import UserModel from '../../model/UserModel';
+import Templates from '../../model/Objects/templates';
+
+function TestButtons() {
+  return (
+    <Grid>
+      <div>Snippet Access</div>
+      <br />
+      <Button onClick={() => SnippetAccessModel.create(['snippetid1', Templates.SnippetAccess])}>
```

Comment:
can we make this `create({ id: 'snippetid1', data: Templates.SnippetAccess })`?

or put `id` with the `data` object.

same comment applies for all fns that has the same implementation as this

Reply:
Is this fine if I'll do this once the UI and db gets integrated? It was coded this way for brevity during testing 😃